lib/model/typing_status.dart

```diff
@@ -38,6 +39,10 @@ class TypingStatus extends ChangeNotifier {
   }
 
   bool _addTypist(SendableNarrow narrow, int typistUserId) {
+    if (typistUserId == selfUserId) {
```

Comment:
commit-message nit:

    model: Do not allow adding self as typist.

For files in lib/model/, we normally make the prefix more specific than this — here, `typing_status:` would be good.

(I'd be tempted to shorten it to `typing:`, but that's too confusing with the concept of the type system and type checker.)